.ci/install_asset.sh

```diff
@@ -17,26 +17,36 @@
 set -e
 
 repo_owner="clearcontainers"
-repo_name="linux"
-cc_linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
 
 function usage() {
 	cat << EOT
-Usage: $0 <version>
-Install the containers clear kernel image <version> from clearcontainers/linux.
+Usage: $0 <kernel|image> <version>
+Install clear containers asset (kernel or image) <version>.
 
-version: Use 'latest' to pull latest kernel or a version from ${cc_linux_releases_url}
+version: Use 'latest' to pull latest asset version
 EOT
 
 	exit 1
 }
 
-function download_kernel() {
-	local version=$1
+
+die(){
+	msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+
+resolve_version() {
+	local repo=$1
+	local version=$2
 	local github_base="https://api.github.com/repos"
-	local latest_release_info_url="${github_base}/${repo_owner}/${repo_name}/releases/latest"
+	local latest_release_info_url="${github_base}/${repo_owner}/${repo}/releases/latest"
+
 	[ -n "${version}" ] || die "version not provided"
+
 	if [ "${version}" == "latest" ]; then
+		echo "latest_release_info_url $latest_release_info_url" >&2
 		release_json="$(curl -s ${latest_release_info_url})"
 		parse_py='
```

Comment:
Reminder to my self, dont use github api to avoid requiere a token.

Reply:
:-) Just to clarify - you can use the ABI without a token, but it is then rate limited, and if a user has burnt up their allowed allocation (as one of my CI systems had), then the call will fail :-)

This PR looks like it has indeed fixed the issue on my CI - so, thanks!